elements/openstack-version-zed.conf.j2

```diff
@@ -0,0 +1,12 @@
+# Include this if you are running zed
+
+[compute]
+# Specify microversions to run tests that depend on new APIs. 2.93 is Zed.
+# See https://docs.openstack.org/nova/latest/reference/api-microversion-history.html
+min_microversion = 2.1
+max_microversion = 2.93
+
+[volume]
+# https://docs.openstack.org/cinder/latest/contributor/api_microversion_history.html#maximum-in-zed-and-2023-1
+min_microversion = 3.0
```

Comment:
```suggestion
min_microversion = 3.44
```
See: https://github.com/openstack/nova/blob/ad15e18ca444ac2ec5553a3585aeb499141e7098/nova/cmd/status.py#L60C28-L60C32

Reply:
I'll leave this as it is not known to cause an issue